testing/deprecated_test.py

```diff
@@ -231,3 +231,31 @@ def test_importing_instance_is_deprecated(pytester: Pytester) -> None:
         match=re.escape("The pytest.Instance collector type is deprecated"),
     ):
         from _pytest.python import Instance  # noqa: F401
+
+
+def test_nose_deprecated(pytester: Pytester) -> None:
```

Comment:
Should also have a test using bare `setup` and `teardown` methods?

Reply:
Glad you asked this, it was not emitting a warning for `setup` and `teardown` 😬